identity-and-access-management/key-manager/api-cli/key-rotation.mdx

```diff
@@ -0,0 +1,105 @@
+publication 800-38D](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-38d.pdf).
+- **Minimize the Impact of Key Compromise:** Regular key rotation limits the number of messages that could be exposed if
+a key is compromised. This reduces the potential damage from such an incident.
+- **Maintain system resilience against security incidents:** Regular key rotation helps your system stay resilient to
+both manual key rotation, whether prompted by a security breach or the need to upgrade to a stronger encryption algorithm.
+- **Regulatory Requirements:** Many industry regulations and standards, such as PCI DSS, NIST guidelines, and others,
+require or recommend regular key rotation as part of maintaining strong cryptographic controls.
+
+## Automated key rotation policy
+
+To configure automatic rotation when creating a key, proceed as follows:
+
+```
+    curl -X POST \
+    --header 'Content-Type: application/json' \
+    --header 'X-Auth-Token: <your_scaleway_token>' \
+    'https://api.scaleway.com/key-manager/v1alpha1/regions/fr-par/keys' \
+    --data '{
+        "project_id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+        "name": "my-key",
+        "usage": {
+            "symmetric_encryption": "aes_256_gcm"
+        },
+        "description": "my key with a rotation policy",
+        "rotation_policy": {
+            "rotation_period": "2592000s", // 30 days
+            "next_rotation_at": "2024-10-01T01:00:00Z"
+        }
+    }'
+```
+
+- **rotation_period:**  duration between two key rotations (min: 24 hours, max: 100 years).
+- **next_rotation_at:** date at which the key will be rotated next.
+
+To configure automatic rotation on an existing key, use the UpdateKey endpoint as follows:
+
+```
+    curl -X PATCH 'https://api.scaleway.com/key-manager/v1alpha1/regions/fr-par/keys/<your_key_id>' \
+    --header 'Content-Type: application/json' \
+    --header 'X-Auth-Token: <your_scaleway_token>' \
+    --data '{
+        "rotation_policy": {
+            "rotation_period": "2592000s", // 30 days
+            "next_rotation_at": "2024-10-01T01:00:00Z"
+        }
+    }'
+```
+
+## Manually rotate your key
+To rotate your key manually, you can use the RotateKey endpoint as shown below:
+
+```
+    curl -X POST 'https://api.scaleway.com/key-manager/v1alpha1/regions/fr-par/keys/<your_key_id>/rotate' \
+    --header 'X-Auth-Token: <your_scaleway_token>' \
+    --data ''
+```
+
+<Message type="important">
+    Avoid relying on irregular or manual key rotation as the primary security measure for your application.
+</Message>
+
+<Message type="note">
+    Manually rotating a key does not interrupt, modify or affect its existing automatic rotation schedule.
+</Message>
+
+<Message type="note">
+    Note that key rotation (both manual and automated) is not possible when you import your own key, because a new key material
+    would be required for each rotation.
```

Comment:
```suggestion
    - Manually rotating a key does not interrupt, modify or affect its existing automatic rotation schedule.
    - Note that key rotation (both manual and automated) is not possible when you import your own key, because a new key would be required for each rotation.
```